lua/hop/defaults.lua

```diff
@@ -1,6 +1,7 @@
 local M = {}
 
 M.keys = 'asdghklqwertyuiopzxcvbnmfj'
+M.quit_key = ''
```

Comment:
Not sure what this means, especially since the doc says it should contain a key with a single character (but the default doesn’t have any?)

Reply:
I should have mentioned in the docs that it can also be an empty string. Then every key gets passed through to neovim as if there was no quit_key. I'll update this.